generators/getMetric.js

```diff
@@ -0,0 +1,68 @@
+const path = require('path')
+const fs = require('fs')
+const objects = require('./getMetricFunctions.js')
+const fsPromises = fs.promises
```

Comment:
is this really needed? Introducing a const only to use it once - you can simply use `fs.promises...`